### PR TITLE
test connection string

### PR DESCRIPTION
**Pull request by krishrustagi**

### CODE REVIEW

File: emotunes-server/src/main/resources/application-prod.yaml [hunk 1]
```diff
@@ -16,6 +16,10 @@ spring:
       max-file-size: 500MB
   kafka:
     bootstrap-servers: localhost:9092
+#<<<<<<< HEAD
+#  azure:
+#    storage:
+#      connection-string: "DefaultEndpointsProtocol=https;AccountName=emotunes;AccountKey=s+BEmpxTFEJZKwpBl5Bo+NdI9zqiwGiQ4fRNOiC7MGJpMLH1/GwfUfj55OqewrJjjBs39G9IFQ6U+AStbmm5cg==;EndpointSuffix=core.windows.net"
```
Comment (SarthakSavasil) [inline emotunes-server/src/main/resources/application-prod.yaml]: same thing commented hai

File: emotunes-server/src/main/resources/application-prod.yaml [hunk 2]
```diff
@@ -16,6 +16,10 @@ spring:
...
 azure:
   storage:
-    connection-string: ${AZURE_STORAGE_CONNECTION_STRING}
```
Comment (SarthakSavasil) [inline emotunes-server/src/main/resources/application-prod.yaml:21]: ye string ese chal nhi rhi thi?

Comment (krishrustagi) [reply emotunes-server/src/main/resources/application-prod.yaml:21]: yeh test branch hai bhai.. deployment ke liye
Chal nhi raha hai isliye test kr raha hu

Comment (krishrustagi) [reply emotunes-server/src/main/resources/application-prod.yaml:21]: Merge nhi krna hai isko

Comment (krishrustagi) [reply emotunes-server/src/main/resources/application-prod.yaml:21]: local par chal raha hai...

**Pull request closed without merging**